pyproject.toml

```diff
@@ -87,3 +87,14 @@ disable = [
     "useless-object-inheritance",
 
 ]
+
+[tool.pytest.ini_options]
+log_cli = true
+# NOTE: If set 'log_cli_level' to 'DEBUG' will show a log of all of the HTTP requests
+# made in functional tests.
+log_cli_level = "INFO"
+log_cli_format = "%(asctime)s.%(msecs)03d [%(levelname)8s] (%(filename)s:%(funcName)s:L%(lineno)s) %(message)s"
+log_cli_date_format = "%Y-%m-%d %H:%M:%S"
+# Force pytest to output color. Without this there is no color when running in Github
+# CI. Even though PY_COLORS=1 is set. Also tried FORCE_COLORS=1.
+addopts = "--color=yes"
```

Comment:
I think I just forgot to add `PY_COLORS` to tox's passenv when doing this, so I think if we don't need it if we add it here:
https://github.com/python-gitlab/python-gitlab/blob/f26bf7d3a86e4d5d1a43423476a46a381e62e8f9/tox.ini#L7

Happens because tox sanitizes the environment I guess, which actually might be very helpful at some point with things like `CI_JOB_TOKEN` ... :grin:

Reply:
Genius! Thanks 👍